install_scripts/templates/replicated-init-aws-ubuntu1404.sh

```diff
@@ -48,8 +48,11 @@ EOF
 
 start_replicated() {
 	mv /etc/replicated-bootstrap/replicated.conf /etc/init
+	mv /etc/replicated-bootstrap/replicated-stop.conf /etc/init
```

Comment:
I've never seen this `/etc/replicated-bootstrap` directory before. What is it?

Reply:
@areed https://help.replicated.com/community/t/creating-an-ami/52/1